db/fixtures/miq_event_definition_events.yml

```diff
@@ -76,10 +76,6 @@
   description: Server High DB Disk Usage
   event_type: Default
   set_type: evm_operations
-- name: evm_server_db_backup_low_space
-  description: Server Database Backup Insufficient Space
-  event_type: Default
-  set_type: evm_operations
```

Comment:
Will this require a data migration to ensure that any existing event defintions aren't tied to any existing objects, or will seed just take care of this?

Reply:
Oh... I don't know... I should probably look into that...

Reply:
I don't know how timelines or other graphs that show events over time would handle a reference to an event definition that no longer exists.   Maybe it stored the definition information whatever it's displaying in the report result itself, similar to how we have a copy of the report definition in each result, in the situation where the report definition changed since it was included.

Reply:
Oh I wasn't even thinking of reports / timelines - I was more thinking things like policies where the event is the driver for the policy.  So we'd have to remove the even from the policy, and then I guess if there are no events left we delete the entire policy.

Reply:
Is it better that we just keep this then?  I just removed it because it had `db_backup` in it, and I don't think anything else was going to be using it.  But if we think it might have lingering uses, I don't mind just keeping it if it makes merging this simpler.

Reply:
If we intend to backport, then we can leave it for this PR, but if not, let's drop it... keeping it around, but having it never fire is worse because it sets up a user expectation that is wrong.

Reply:
I think I will remove this for now, but immediately open up a PR to fix this once it is merged so we can get most of this rolling, since I am unclear about what all is necessary, and I don't want this one thing holding things up.

Reply:
@Fryguy Here is the follow up so it is on our radar:  https://github.com/ManageIQ/manageiq/pull/21430